mappings/net/minecraft/client/gui/screen/TooltipComponent.mapping

```diff
@@ -0,0 +1,13 @@
+CLASS net/minecraft/class_5684 net/minecraft/client/gui/screen/TooltipComponent
+	METHOD method_32661 getWidth ()I
+	METHOD method_32662 createOrderedTextTooltipComponent (Lnet/minecraft/class_5481;)Lnet/minecraft/class_5684;
+	METHOD method_32663 createTooltipComponent (Lnet/minecraft/class_5632;)Lnet/minecraft/class_5684;
```

Comment:
This creates what you called a BundleTooltipComponent

Reply:
Yes but the structure of the method suggests it's for any generic `TooltipData` and would be used if mojang implemented any more.